### PR TITLE
Install standalone lib should fix #169

### LINKED ISSUE #169: error while loading shared libraries: libpsen_scan_v2_standalone.so
### Commit
0.1.5

### Steps to reproduce
On a clean system do 
```
sudo apt get install ros-noetic-psen-scan-v2
```

### Expected behavior
Node starts successfull

### Observed behavior
```
/opt/ros/noetic/lib/psen_scan_v2/psen_scan_v2_node: error while loading shared libraries: libpsen_scan_v2_standalone.so: cannot open shared object file: No such file or directory
process[robot_state_publisher-3]: started with pid [18752]
process[rviz-4]: started with pid [18753]
================================================================================REQUIRED process [laser_scanner-2] has died!
process has died [pid 18751, exit code 127, cmd /opt/ros/noetic/lib/psen_scan_v2/psen_scan_v2_node __name:=laser_scanner __log:=/home/agutenkunst/.ros/log/13fb5334-8583-11eb-9c8d-4b7c32c1d81c/laser_scanner-2.log].
log file: /home/agutenkunst/.ros/log/13fb5334-8583-11eb-9c8d-4b7c32c1d81c/laser_scanner-2*.log
Initiating shutdown!
================================================================================
```

### PR DESCRIPTION
**Pull request by rfeistenauer**
## Description

These changes fix #169 by installing the standalone lib in the ros environment

### Things to add, update or check by the maintainers before this PR can be merged.

* [x] ~Public api function documentation~
* [x] ~Architecture documentation reflects actual code structure~
* [x] ~[Tutorials](https://wiki.ros.org/pilz_robots/Tutorials/)~
* [x] ~Overview on [ROS wiki](https://wiki.ros.org/pilz_robots)~
* [x] ~Package Readme ([example pilz_robots](https://github.com/PilzDE/pilz_robots/blob/melodic-devel/README.md))~
* [x] Good commit messages ([some tips](https://dev.to/jacobherrington/how-to-write-useful-commit-messages-my-commit-message-template-20n9))
* [x] CHANGELOG.rst updated
* [x] ~Copyright headers~
* [x] ~Examples~

### Review Checklist
* [x] ~Soft- and hardware architecture (diagrams and description)~
* [x] Test review (test plan and individual test cases)
* [x] ~Documentation describes purpose of file(s) and responsibilities~
* [x] Code (coding rules, style guide)

### Release planning (please answer)
* [x] When is the new feature released? => ASAP
* [x] ~Which dependent packages do have to be released simultaneously?~

### CODE REVIEW

**Comment by rfeistenauer** [comment]
acording to:
https://stackoverflow.com/questions/34443128/cmake-install-targets-in-subdirectories

solutions are:
- add install(Targets ...standalone) in ROS CMakeLists which requires Cmake 3.13 which does not work with melodic
- move add_library to ROS CMakeLists and dont use subdirectory which is more or less copy past of all standalone stuff
- add install(Targets ...standalone) to standalone CMakeLists but in melodic LIBRARY DESTINATION parameter is not optional so it would have to be cakin_LIBRARY_DESTINATION in ROS and something else in standalone builds

File: CMakeLists.txt [hunk 1]
```diff
@@ -76,22 +76,43 @@ if(CATKIN_ENABLE_CLANG_TIDY)
...
+  standalone/src/data_conversion_layer/monitoring_frame_msg.cpp
+  standalone/src/data_conversion_layer/start_request.cpp
+  standalone/src/data_conversion_layer/start_request_serialization.cpp
+  standalone/src/data_conversion_layer/stop_request_serialization.cpp
+  standalone/src/data_conversion_layer/monitoring_frame_deserialization.cpp
+  standalone/src/data_conversion_layer/diagnostics.cpp
+  standalone/src/data_conversion_layer/scanner_reply_serialization_deserialization.cpp
+)
+
+add_library(${PROJECT_NAME}_standalone ${${PROJECT_NAME}_standalone_sources})
```
Comment (agutenkunst) [inline CMakeLists.txt:103]: please use linebreaks

**Review by agutenkunst — APPROVED** [review]
Just one small suggestion

File: CMakeLists.txt (hunk 1 above)
Comment (rfeistenauer) [inline CMakeLists.txt:103]:
```suggestion
add_library(
  ${PROJECT_NAME}_standalone 
  ${${PROJECT_NAME}_standalone_sources}
)
```

**Pull request merged**